### ISSUE TITLE
Required Repeating Section

### ISSUE THREAD

**Issue by calexh-sar** [sartography/cr-connect#372]
Need a way to specify that completion of at least one Repeating Section is required.  The Form Save button should be Inactive until at least one Repeating Section has been saved.  This is needed in the case of required document which allows for multiple documents of that type to be uploaded.

![image.png](https://images.zenhubusercontent.com/5f0623d8a1d620a9f39301dc/06ca59e1-8476-4c63-8bbc-a28b59e51eb8)

**Comment by Leashys** [comment]
alex - -I think there is already functionality for this if you add this constraint:

![Screenshot 2021-07-09 at 16-05-59 Research Ramp-Up Toolkit Configurator.png](https://images.zenhubusercontent.com/605376307bf33b5036eb360b/93faeb26-3727-4946-b866-1abc133ef5ca)

**Comment by calexh-sar** [comment]
@Leashys I just confirmed that the repeat_required  constraint is available and working, i.e., the Save button is disabled.  The availability of this constraint was not documented in the Configurator User Guide.  I added it here. 

Will write another ticket and assign to you to get the message related to it improved:

![image.png](https://images.zenhubusercontent.com/5f0623d8a1d620a9f39301dc/200c34b9-9b83-4bac-9ca2-71bf6e8b8ad8)

The primary reason we had to get you to dig into the code and see if this functionality was available is it was not documented in the Configurator User Guide.  I added it [here](https://github.com/sartography/cr-connect-bpmn/wiki/CR-Connect-Workflow-Configurator---User-Guide#making-a-repeating-section-required).  Ping me when you have a moment if you have not been brought up to speed about the Configurator User Guide and keeping it up to date.
